src/app/analyze/stack/stack-details/add-work-flow.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { Http, Response, Headers, RequestOptions } from '@angular/http';
+
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/catch';
+import 'rxjs/add/operator/map';
+
+@Injectable()
+export class AddWorkFlowService {
+
+  private stackWorkItemUrl = 'http://demo.api.almighty.io/api/workitems';
+  private authoBearerToken = `Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJmdWxsTmFtZSI6ImphaXZhcmRoYW4gS3VtYXIiLCJpbWFnZVVSTCI6Imh0dHBzOi8vd3d3LmdyYXZhdGFyLmNvbS9hdmF0YXIvOTY4YWZjOTgwY2YyZjE0NzVjNTExMmM4ZGYwMjhhNTIuanBnIiwidXVpZCI6ImU4MmRlNzc1LWRhNjQtNDIwMi04MTYzLTAyMWJhNjU1MzI3ZCJ9.YSl4FvcfXw0r2PfZ6pIj8SRZtMX_CtmxDHaM439RdaaZmeDzfc90IeU1UaP4-Sgz_onAp2BPrczxVFFSa_dwdwi7NRV6BTAtBdMYldH7VTsgeQlNUNcUuX6Pzj4K45jiB3Y367eWILniHKXmLdA0qZPO9R5Cd5Gu44TfMNM2SIJiQXArGF9FaQ1TWTS9QVQkm7yQ7a6qxlRMdBLPWKp0OSyF--BtI7kWAJj83opHwaLvh8EZZRoRsy9yBefpzU2c2odbI3sb137jic7HCPq_c8M7YTdXr93IEyodGVoLRKIPw6mGBShCo9wIWZ7u5aNcuxis6VcEvO_OhYxnr33BAw`;
```

Comment:
I've passed on a token generated locally for 'testuser' in devmode to @invincibleJai .
That should be fine if public, because the corresponding secrets needed are also in the core public repo ;)